CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ Canonical reference for changes, improvements, and bugfixes for Boundary.
   setting this environment variable would result in a max retries of 2,
   regardless of the value set.
   ([PR](https://github.com/hashicorp/boundary/pull/5385)).
+* Fix bug in parsing IPv6 addresses. Previously setting a target address and the
```

Comment:
I'm a little confused about the wording here.
- Does the problem occur when you set happen to set both a "target address AND an initial upstream address" INSIDE a config file 
- Or is the problem with setting either a "target address" OR an "initial upstream address in the config file"